### PR TITLE
chore: migrate to vite

### PR DESCRIPTION
**Pull request by ReidyT**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,32 +36,35 @@
     "react-dom": "18.2.0",
     "react-i18next": "12.3.1",
     "react-markdown": "^8.0.7",
-    "react-scripts": "5.0.1",
     "react-toastify": "9.1.3",
     "remark-breaks": "^3.0.3",
+    "remove": "^0.1.5",
```
Comment (spaenleh) [inline package.json:41]: What is this ?

File: tsconfig.json [hunk 2]
```diff
@@ -11,13 +11,22 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "ESNext",
+    "moduleResolution": "Node",
```
Comment (spaenleh) [inline tsconfig.json:15]: Use "bundler" instead ?

**Review by spaenleh — APPROVED** [review]
Looks good !

**Comment by spaenleh** [comment]
Can you deploy it on dev to test ?

File: package.json (hunk 1 above)
Comment (ReidyT) [reply package.json:41]: I think its a mistake on my side

**Review by spaenleh — APPROVED** [review]
🚀

**Pull request merged**